src/metabase/task/sync_databases.clj

```diff
@@ -9,7 +9,7 @@
             [metabase
              [task :as task]
              [util :as u]]
-            [metabase.models.database :refer [Database]]
+            [metabase.models.database :as mdb :refer [Database]]
```

Comment:
the convention is that we alias models as `database`. Makes things more grep-able if we are consistent.